### ISSUE TITLE
Issue parsing printed GraphQL schema (documentation that ends in `"`)

### ISSUE THREAD

**Issue by benjie** [graphql/graphql-js#1337]
If a description in a GraphQL schema ends with a quotation mark `"` then you can no longer `parse(printSchema(schema))` - i.e. the printed schema cannot be parsed.

Here's a small script that reproduces the issue:

```js
const {
  GraphQLObjectType,
  GraphQLSchema,
  GraphQLInt,
  printSchema,
  parse,
} = require("graphql");

const makeSchema = withBadComment =>
  new GraphQLSchema({
    query: new GraphQLObjectType({
      name: "Query",
      fields: {
        hello: {
          type: GraphQLInt,
        },
      },
      description: withBadComment
        ? `Something ending with a "quote"`
        : `Something with "quote" inside.`,
    }),
  });

function test(schema) {
  const schemaString = printSchema(schema);
  console.log(schemaString);

  const parsed = parse(schemaString);
  console.dir(parsed);
}

console.log("Testing good schema");
test(makeSchema(false));

console.log();
console.log();
console.log(
  "********************************************************************************"
);
console.log();
console.log();
console.log("Testing bad schema");
test(makeSchema(true));
```

GraphQL version: 0.12.3

**Comment by benjie** [comment]
(This is solved in 0.13.2 but we can't use that because https://github.com/graphql/codemirror-graphql still requires 0.12. )

**Comment by IvanGoncharov** [comment]
@benjie Yes you right it was fixed in #1205
We are working on `14.0.0` that will include this fix.

**Comment by benjie** [comment]
Is it possible to either get a fix for this backported to 0.12.x, or have codemirror-graphql support 0.13.2+?

**Comment by IvanGoncharov** [comment]
> (This is solved in 0.13.2 but we can't use that because graphql/codemirror-graphql still requires 0.12. )

@benjie Sorry, I completely missed that comment. Just trying to push `14.0.0` forward by cleaning up issues/PRs.

> codemirror-graphql support 0.13.2+?

Codemirror depends on [graphql-language-service](https://github.com/graphql/graphql-language-service) so it should be updated first and I don't have commit rights to this repo.
I will try to make PR this week and ping someone who has commit rights.
Feel free to ping me in this issue.
